runtime/lang/menu_fr_fr.latin1.vim

```diff
@@ -1,486 +1,1267 @@
-" Menu Translations:	Francais
-" Maintainer:		Adrien Beau <version.francaise@free.fr>
-" First Version:	Francois Thunus <thunus@systran.fr>
-" Contributors:		David Blanchet <david.blanchet@free.fr>
-" 			Gabriel Dupras <gabrieldupras@gmail.com>
-" Last Change:		2021 Feb 16
+" Menu Translations:	  Francais
+" Maintainer:           Ada (Haowen) Yu <me@yuhaowen.com>
+" Previous Maintainer:  Adrien Beau <version.francaise@free.fr>
+" Contributors:         Francois Thunus <thunus@systran.fr>, David Blanchet <david.blanchet@free.fr>, Gabriel Dupras <gabrieldupras@gmail.com>
+" Last Change:          2022 July 10
 " Original translations
+"
+" Generated with the scripts from:
+"
+"       https://github.com/adaext/vim-menutrans-helper
 
 " Quit when menu translations have already been done.
+
 if exists("did_menu_trans")
   finish
 endif
 let did_menu_trans = 1
-let s:keepcpo= &cpo
+let s:keepcpo = &cpo
 set cpo&vim
 
-" The translations below are in latin1, but they work for cp1252 and
-" iso-8859-15 without conversion as well.
-if &enc != "cp1252" && &enc != "iso-8859-15" && &enc != "iso-8859-1"
-  scriptencoding latin1
+scriptencoding latin1
+
+" Help menu
+menutrans &Help &Aide
+" Help menuitems and dialog {{{1
+menutrans &Overview<Tab><F1> &Sommaire<Tab><F1>
+menutrans &User\ Manual &Manuel\ utilisateur
+menutrans &How-to\ Links &T�ches\ courantes
```

Comment:
`scriptencoding` is set to "latin1", but actually this file is saved with UTF-8 with many U+FFFD marks (replacement character), which means that this file is broken.